packages/lib-classifier/src/store/utils/ClassificationQueue.js

```diff
@@ -3,15 +3,17 @@
 // Also for browsers that do not support Background Sync API
 
 import { panoptes } from '@zooniverse/panoptes-js'
+import { getBearerToken } from './'
 
 const FAILED_CLASSIFICATION_QUEUE_NAME = 'failed-classifications'
 const MAX_RECENTS = 10
 const RETRY_INTERVAL = 5 * 60 * 1000
 
 class ClassificationQueue {
-  constructor (api, onClassificationSaved) {
+  constructor (api, onClassificationSaved, authClient) {
     this.storage = window.localStorage
     this.apiClient = api || panoptes
+    this.authClient = authClient
```

Comment:
Do we want a default for the auth client here? If so, what should it be?

Reply:
I don't think we do - we should add a console statement saying there is no auth client, but we shouldn't require one / default to one.

Reply:
Authorization headers aren't required on classifications, so I'm happy to default this to `undefined`.